src/testRunner/unittests/tsserver/projects.ts

```diff
@@ -708,7 +708,7 @@ namespace ts.projectSystem {
             // Check identifiers defined in HTML content are available in .ts file
             const project = configuredProjectAt(projectService, 0);
             let completions = project.getLanguageService().getCompletionsAtPosition(file1.path, 1, emptyOptions);
-            assert(completions && completions.entries[0].name === "hello", `expected entry hello to be in completion list`);
+            assert(completions && completions.entries[1].name === "hello", `expected entry hello to be in completion list`);

```

Comment:
What became the 0th completion?

Reply:
It's globalThis. Hm. I'll add the assertion although it's not very interesting. I'll make a note in the assertion message.